test cases/rust/20 object link/meson.build

```diff
@@ -0,0 +1,22 @@
+project('staticprog', 'c')
+
+rustc = find_program('rustc')
+add_languages('rust')
+
+# Requires a discoverable std library, most likely via pkg-config,
+# but there is no common standard so make it optional for now.
+libstd_rust = dependency('std-rust', required: false)
```

Comment:
What I think this means is that it's totally untested in CI?

Instead of checking if this is found and then making an `executable()`, this should raise the error `MESON_SKIP_TEST: <explanation>` if not found. That will cause the test to properly log as skipped.

Reply:
The linkage is skipped, but building the object (which is what this PR enables) is always done. Changed to skip.